tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Shape.cpp

```diff
@@ -430,38 +434,35 @@ SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.TimeDependent.Shape",
   domain::CoordinateMaps::ShapeMapTransitionFunctions::
       register_derived_with_charm();
 
+  const CoordinateMaps::ShapeMapTransitionFunctions::SphereTransition
+      sphere_transition{2., 100.};
   MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<double> dist_coords{2., 10.};
```

Comment:
Why are you restricting to the first octant only now?

Reply:
The SphereTransition requires the coords are within r_min and r_max. Before this commit, this was "solved" by setting r_min to 1e-7, which is unlikely to be hit by the random number generator.

Reply:
I don't see how your comment answers Kyle's question.  You've changed the test so that all test points have positive `(x,y,z)`

Reply:
Right, this seemed the easiest way to solve the issue I described. Do you want me to change it somehow?

Reply:
Ah okay, I didn't understand the issue as I thought the ShapeMap would have been defined everywhere.  This should be made explicit in the ShapeMap if it is only valid between `[r_min, r_max]`.  

So I would have the test pick 
- a random center with (x_c, y_c, z_c) allowed positive or negative
- a random radius in [r_min, r_max]
- a `random_unit_normal`
and compute the random point from them

Reply:
might be a useful helper function `random_point_in_shell(r_min,r_max,center)`

Reply:
Ok yeah that explains it. I like @kidder's suggestion for the test.

Reply:
Sorry I don't have time for this at the moment, I just dropped this commit. It's only the bugfix commit now.